stack-graphs/Cargo.toml

```diff
@@ -13,11 +13,13 @@ authors = [
 edition = "2018"
 
 [features]
+default = ["tree-sitter"]
 bincode = ["dep:bincode", "lsp-positions/bincode"]
 copious-debugging = []
 serde = ["dep:serde", "serde_with", "lsp-positions/serde"]
 storage = ["bincode", "rusqlite"]
 visualization = ["serde", "serde_json"]
+tree-sitter = ["lsp-positions/tree-sitter"]
```

Comment:
So this one is also not necessary. Clients that want the `tree-sitter` functionality can do that by explicitly depending on `lsp-positions` and enabling the features.